var/spack/repos/builtin/packages/fio/package.py

```diff
@@ -32,6 +33,9 @@ class Fio(AutotoolsPackage):
     conflicts('+libaio', when='platform=darwin',
               msg='libaio does not support Darwin')
 
+    conflicts('@:3.18', when='%gcc@10:',
+            msg='gcc@10: sets -fno-common by default')
```

Comment:
```suggestion
              msg='gcc@10: sets -fno-common by default')
```

Reply:
Thanks and fixed! I should really set up a flake8 post-commit hook. :laughing: